build-support/ivy/ivy.xml

```diff
@@ -18,8 +18,8 @@ limitations under the License.
 -->
 
 <ivy-module version="2.0"
-            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-            xsi:noNamespaceSchemaLocation="http://ant.apache.org/ivy/schemas/ivy.xsd">
+            xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance"
+            xsi:noNamespaceSchemaLocation="https://ant.apache.org/ivy/schemas/ivy.xsd">
```

Comment:
This seems to break CI with:
```
 Exception in thread "main" java.text.ParseException: [xml parsing: ivy.xml:22:88: cvc-complex-type.3.2.2: Attribute 'xsi:noNamespaceSchemaLocation' is not allowed to appear in element 'ivy-module'. in file:/home/travis/build/twitter/commons/build-support/ivy/ivy.xml
 ]
 	at org.apache.ivy.plugins.parser.AbstractModuleDescriptorParser$AbstractParser.checkErrors(AbstractModuleDescriptorParser.java:88)
 	at org.apache.ivy.plugins.parser.AbstractModuleDescriptorParser$AbstractParser.getModuleDescriptor(AbstractModuleDescriptorParser.java:342)
 	at org.apache.ivy.plugins.parser.xml.XmlModuleDescriptorParser$Parser.getModuleDescriptor(XmlModuleDescriptorParser.java:180)
 	at org.apache.ivy.plugins.parser.xml.XmlModuleDescriptorParser.parseDescriptor(XmlModuleDescriptorParser.java:120)
 	at org.apache.ivy.plugins.parser.AbstractModuleDescriptorParser.parseDescriptor(AbstractModuleDescriptorParser.java:48)
 	at org.apache.ivy.core.resolve.ResolveEngine.resolve(ResolveEngine.java:187)
 	at org.apache.ivy.Ivy.resolve(Ivy.java:508)
 	at org.apache.ivy.Main.run(Main.java:311)
 	at org.apache.ivy.Main.main(Main.java:219)
```

Switching to coursier might be an option to resolve this one?

Reply:
@stuhood I'm not quite sure how to make this project use coursier, would you be willing to lend me a hand?